domains/lullu.json

```
@@ -0,0 +1,9 @@
+{
+	"owner": {
+		"username": "esc5221",
+		"email": "esc5221@gmail.com"
+	},
+	"record": {
+		"CNAME": "esc5221.github.io/blog"
```

CNAME can't contain `/`
```suggestion
		"CNAME": "esc5221.github.io"
```